src/error-table.rkt

```diff
@@ -0,0 +1,49 @@
+#lang racket
+
+(require "points.rkt" "syntax/types.rkt" "core/localize.rkt" "timeline.rkt")
+
+(provide group-errors)
+
+(define (zip l1 l2) (map list l1 l2))
+
+(define (group-errors expr pcontext)
+  (define points (for/list ([(pt _) (in-pcontext pcontext)]) pt))
+  (define local-errors (car (compute-local-errors (list expr) (*context*))))
+
+  (define max-per-point (make-hash))
+  (define errpts-per-subexpr (make-hash))
+
+  (for ([pt points])
+    (hash-set! max-per-point pt (list 0 0)))
+
+  (for ([(sub-expr errors) local-errors])
+    (hash-set! errpts-per-subexpr sub-expr null)
+    (for ([pair (zip points errors)])
+      (define pt (car pair))
+      (define err (cadr pair))
```

Comment:
Prefer `match-define` in cases like this:

```
(match-define (list pt err) pair)
```

But actually here constructing `pair` is needless to begin with:

```
(for ([pt (in-list points)] [err (in-list errors)])
  ...)
```